src/Components/DataDisplay/Tabs.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+	<swiper v-if="type === 'swiper'" ref="mySwiper" class="swiper" :options="swiperOptions">
+		<swiper-slide v-for="(item, index) in items" :key="`tabs-item-${item}-${index}`">
+			<button class="c-tabs--menu-button" :class="{ active: index === selectedNo }" @click="setSelectedNo(index)">
+				{{ item }}
+			</button>
+		</swiper-slide>
+	</swiper>
+	<div v-else class="c-application c-tabs--menu-wrapper">
```

Comment:
```suggestion
	<div v-else class="c-application c-tabs--menu-container">
	    <div class="c-tabs--menu-wrapper">
```
보통 container > wrapper 순으로 사용하니까 클래스명을 바꾸면 좋을 것 같네요.